org.virt_manger.virt-viewer.appdata.xml

```diff
@@ -0,0 +1,60 @@
+<!-- https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html -->
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
```

Comment:
No screenshots.

Reply:
What resolution should screenshots be? As a reference.